packages/paste-design-tokens/tokens/themes/sendgrid/global/background-color.yml

```diff
@@ -84,13 +84,13 @@ props:
 
   # destructive backgrounds
   color-background-destructive-darker:
-    value: "{!palette-red-90}"
+    value: "{!sg-danger-darker}"
```

Comment:
For both dark/darker we're using the same values of curious-blue and sg-danger-darker. Is that expected since we only really have a primary and one dark shade in SG colors for primary and danger?

Reply:
I did this because in our design language we only have one intensity/hue/lightness value and it doesn't map 1:1 to our components which was part of the intention for this. I think Sean/Paste Team need to decide what the appropriate strategy is.